nmtwizard/preprocess/tu.py

```diff
@@ -330,10 +364,25 @@ def __init__(
         alignment_log_probs=None,
         source_tokenizer=None,
         target_tokenizer=None,
+        source_context=None,
+        target_context=None,
```

Comment:
What is the usage of a target context? Shouldn't the target be the translation of the main source?

Reply:
I wanted to test both approaches : the one with the translation of the main source as target, and the one with multi-sentence target (where only the last sentence will be used for evaluation).